src/sig-verifier-js/tests.ts

```diff
@@ -1,18 +1,142 @@
+  }
+],
+    "publicKey":"303c300c060a2b0601040183b8430102032c000a00000000006000270101f3ffab2278616508ad5ebfa0cb79a21e08dbb7132f6875b95f81e72067f31302"
+}`;
+// Equal to delegation.expiration-field in DELEGATION_CHAIN_JSON.
+const EXPIRATION: bigint = BigInt(1708469015156620577);
+// Id of the II-instance that generated DELEGATION_CHAIN_JSON.
+const II_CANISTER_ID: string = "fgte5-ciaaa-aaaad-aaatq-cai";
+// User identity as viewed by the FE that called the II-instance to get DELEGATION_CHAIN_JSON.
+const USER_PRINCIPAL_ID: string =
+  "hf7wk-a35mp-bc6eb-ntvr2-aeu3d-naglw-n6ea3-qn5ps-jcanu-p2vro-5ae";
+
+// Equal to delegation.pubkey-field in DELEGATION_CHAIN_JSON
+const CHALLENGE = new Uint8Array([
+  0xe7, 0x87, 0x5e, 0x69, 0xce, 0x7b, 0xed, 0xa6, 0xfc, 0x7b, 0x6d, 0xfb, 0xd9,
+  0xb7, 0x5b, 0xe1, 0xc6, 0xf6, 0xd5, 0xde, 0xba, 0xe3, 0xae, 0x1e, 0xd7, 0xc7,
+  0xf8, 0x73, 0xde, 0x1b, 0x6f, 0x9f, 0x75, 0xe9, 0xe7, 0xdc, 0xdd, 0xcf, 0x37,
+  0xef, 0xad, 0xdc, 0xdf, 0x6f, 0x7b, 0x69, 0xa7, 0xb5, 0x73, 0x77, 0xb5, 0xdd,
+  0xae, 0xf8, 0x7d, 0xee, 0x38, 0x6d, 0xdd, 0x75, 0xe3, 0x9e, 0x9c, 0xd3, 0x9d,
+  0x7d, 0x77, 0xde, 0xbc, 0x79, 0xdf, 0x1b, 0x7b, 0x46, 0x9d, 0xf3, 0x6e, 0xb8,
+  0xe7, 0xce, 0xf4, 0x7b, 0x4d, 0x5c, 0xef, 0xa7, 0xf5, 0xdf, 0x67, 0xdb, 0xef,
+  0xc7, 0x3d, 0xeb, 0xdf, 0x5c,
+]);
+
+export var ROOT_PUBLIC_KEY_RAW = new Uint8Array([
```

Comment:
please use `const` when possible. Also I think you might be mutating this  in `Should fail validateDelegationAndGetPrincipal with wrong IC root public key` by settings the 42nd byte. Rearranging the tests might surface the issue.

Reply:
changed to `const`